coops-ui/public/config/configuration.json

```diff
@@ -2,6 +2,7 @@
   "API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/businesses/",
   "AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1/",
   "PAY_API_URL": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1/",
+  "KEYCLOAK_CONFIG_URL": "/local-keycloak-config-url/keycloak.json",
```

Comment:
The value above is for running locally only (see also vue.config.js below). When deployed, the OpenShift config map should contain a line like this:
`"KEYCLOAK_CONFIG_URL": "https://dev.bcregistry.ca/cooperatives/auth/config/kc/keycloak.json"`